scripts/SimplePyQtGUIKit.py

```diff
@@ -80,10 +83,10 @@ def GetCheckButtonSelect(self, selectList, title="Select", msg="",app=None):
         return result
 
 if __name__ == '__main__':
-    #  print "GetCheckButtonSelect"
+    #  print("GetCheckButtonSelect")
     #  optList=SimplePyQtGUIKit.GetCheckButtonSelect(["sample a","sample b","sample c"], title="Select sample", msg="Please select sample")
-    #  print optList
+    #  print(optList)
```

Comment:
```suggestion
```

Reply:
fixed: https://github.com/AtsushiSakai/rosbag_to_csv/pull/14/commits/236078068f6b8a4a565010d0a22b0b09a95a6873 
Also, removed `optList` variable in line 87